lib/matplotlib/collections.py

```diff
@@ -889,18 +932,24 @@ def draw(self, renderer):
 
 class PathCollection(_CollectionWithSizes):
     """
-    The most basic `Collection` subclass, created e.g. by `~.Axes.scatter`.
+    The most basic `.Collection` subclass, created e.g. by `~.Axes.scatter`.
     """
 
-    @docstring.dedent_interpd
     def __init__(self, paths, sizes=None, **kwargs):
         """
-        *paths* is a sequence of `matplotlib.path.Path` instances.
-
-        %(Collection)s
+        Parameters
+        ----------
+        paths : list of matplotlib.path.Path
+            The paths that will make up the `.Collection`.
+        sizes : array-like of float
```

Comment:
```suggestion
        sizes : array-like
```
https://matplotlib.org/devdocs/devel/documenting_mpl.html#parameter-type-descriptions
> float is the implicit default dtype for array-likes. For other dtypes use array-like of [type].